sections/05_treat.md

```diff
@@ -78,6 +78,53 @@ able to significantly outperform a random forest classifier trained on gene
+more traditional drug discovery approaches [@doi:10.1016/j.jhealeco.2016.01.012
+@doi:10.1038/nrd4609]. Ever since the concept of the Connectivity Map was
+proposed [@doi:10.1126/science.1132939], several advanced computational methods
+have been applied to formulate and validate drug repositioning hypotheses
+[@doi:10.1093/bib/bbv020 @doi:10.1093/bib/bbw112 @doi:10.1093/bib/bbw110]. Using
+supervised learning and recommender systems to tackle this type of problems is
+proving successful in different scenarios, especially when coupled with
+topological information from protein - protein or protein - compound interaction
+networks [@doi:10.1186/1758-2946-5-30 @doi:10.1021/ci500340n
+@doi:10.1186/s12859-015-0845-0]. Menden et al
+[@doi:10.1371/journal.pone.0061318] used a shallow neural network to predict
+sensitivity of cancer cell lines to drug treatment using both cell line and drug
+features, opening the door to precision medicine and drug repositioning
+opportunities in cancer. More recently, Aliper et al
+[@doi:10.1021/acs.molpharmaceut.6b00248] used gene- and pathway-level drug
+perturbation transcriptional profiles from the Library of Network-Based Cellular
+Signatures (LINCS) [@doi:10.3389/fgene.2014.00342] to train a fully connected
+deep neural network able to predict drug therapeutic uses and indications. By
+using confusion matrices and leveraging misclassification, the authors formulate
+a number of interesting hypotheses, including repurposing cardiovascular drugs
+such as otenzepad and pinacidil for neurological disorders.
+
+Drug repositioning can also be approached by attempting to predict novel drug -
+target interactions and then repurposing the drug for the associated
+indication [doi:10.1371/journal.pcbi.1005219 @doi:10.1371/journal.pcbi.1005135].
+Wang et al [@doi:10.1109/BIBM.2014.6999129] devised a pairwise input neural
+network with two hidden layers that takes two inputs, a drug and a target
+binding site, and predicts whether they interact. Wang et al
+[@doi:10.1093/bioinformatics/btt234] trained individual RBMs for each target in
+a drug - target interaction network and used these models to predict novel
+interactions pointing to new indications for existing drugs. Wen et al.
+[@doi:10.1021/acs.jproteome.6b00618] extended this concept to
+deep learning by creating a deep belief network of stacked RBMs, DeepDTIs,
+predicting interaction on the basis of chemical structure and protein sequence
+features.
+
+Drug repositioning appears to be an obvious candidate for the development of
+deep learning applications both because of the large amount of high-dimensional
+data available and because of the complexity of the question being asked.
+However, we believe these are still very early days and what is perhaps the most
+promising piece of work in this space [@doi:10.1021/acs.molpharmaceut.6b00248]
+is more a proof of concept than a real-world hypothesis-generation tool.
+
```

Comment:
Based on the unmodified text below, you might be working from an older version of the master branch.  This should be okay as long as you don't edit anything in the other sub-sections of Treat.  If you need to do that, perhaps wait until we merge this.